CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing Pull Requests
```

Comment:
@AaronCrawfis Copied most of it from the Radius repo contributions guide. Let me know what you think.

Reply:
Thanks for noticing that we're missing a CONTRIBUTING.md file! We'll definitely need one. Could you actually just make this file a pointer to https://docs.radapp.io/community/contributing/docs/ ?

We have most of the contributing information there.

Thanks!!

Reply:
Thought I was in the docs repo, please disregard 😆